config.template.py

```diff
@@ -19,6 +19,12 @@
 SECURITY_REGISTERABLE = True
 SECURITY_PASSWORD_SALT = "some_salt"
 
+# CONFIG USED FOR FLASK-PRAETORIAN
```

Comment:
Nice setup, good names and defaults!

Reply:
As I remember we did this together, thanks for the help :)